songr/src/main/java/com/cfjava/songr/controllers/songController.java

```diff
@@ -0,0 +1,45 @@
+package com.cfjava.songr.controllers;
+
+import com.cfjava.songr.models.Album;
+import com.cfjava.songr.models.Song;
+import com.cfjava.songr.repositories.AlbumsRepository;
+import com.cfjava.songr.repositories.SongRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.servlet.view.RedirectView;
+
+import java.util.List;
+
+@Controller
+public class songController {
```

Comment:
Best practice: the names (and file names) of Java classes should be capitalized (SongController, AlbumController, etc.). That prevents Java from getting confused over what is a class and what is a variable.